components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -113,7 +114,8 @@ - (instancetype)initWithTitle:(NSString *)title message:(NSString *)message {
     _header = [[MDCActionSheetHeaderView alloc] initWithFrame:CGRectZero];
     _header.title = [title copy];
     _header.message = [message copy];
-    self.backgroundColor = [UIColor whiteColor];
+    _backgroundColor = UIColor.whiteColor;
```

Comment:
Do you need to set the `backgroundColor` values for `view`, `header`, and `tableView`?

Reply:
I'm confused by this change. It looks like you're no longer setting the view's `backgroundColor` property, is that correct? Should these views initialize without a background color? Do `_header` and `_tableView` no longer require background colors either?

Reply:
I am now applying background color to the `view`, `_header`, and `_tableView`.